repos/system_upgrade/el7toel8/actors/commonleappdracutmodules/files/dracut/85sys-upgrade-redhat/do-upgrade.sh

```diff
@@ -121,6 +139,9 @@ result=$?
 ##### safe the data and remount $NEWROOT as it was previously mounted #####
 save_journal
 
+#FIXME: for debugging purposes; this will be removed or redefined in future
```

Comment:
@vinzenz do you want to remove it in this PR or later?

Reply:
It's the same comment on the other places, I just duplicated it as well - I want to leave it there.